ext/RMagick/rmagick_gvl.h

```diff
@@ -0,0 +1,224 @@
+#ifndef _RMAGICK_GVL_H_
+#define _RMAGICK_GVL_H_
```

Comment:
What does this define do?

Reply:
Thanks. I found the miss in endif position.
This allow to include this header file at many time.